qiskit_experiments/analysis/plotting.py

```diff
@@ -81,18 +88,24 @@ def plot_curve_fit(
     ax.plot(xs, ys_fit, **plot_opts)
 
     # Plot standard error interval
-    if confidence_interval and fit_errors is not None:
+    if monotonicity and fit_errors is not None:
         if param_keys:
             params_upper = {}
             params_lower = {}
             for key, param, error in zip(param_keys, fit_params, fit_errors):
-                params_upper[key] = param + error
-                params_lower[key] = param - error
+                params_upper[key] = param - error * (-1) ** monotonicity[key]
```

Comment:
This is a somewhat confusing hack since it uses boolean values in a power operation (possible in Python, but doesn't make the code more readable).

Reply:
Do you have a suggestion?

Reply:
You can use the ternary operator, something like

```
error_coeff = 1 if is_increasing[key] else -1
params_upper[key] = param + (error_coeff) * error
params_lower[key] = param - (error_coeff) * error
```